ml-algorithms/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
        exclude group: "org.jetbrains", module: "annotations"
     }
     implementation group: 'org.apache.commons', name: 'commons-text', version: '1.10.0'
+    implementation group: 'com.github.erosb', name: 'json-sKema', version: '0.15.0'
```

Comment:
Why don't we use https://github.com/json-schema-org/json-schema-spec?
The package "com.github.erosb.json-sKema" looks like having very few references, not sure if it is prod ready.

Reply:
Changed to top used library.